src/modules/eventRegister/info-banner/InfoBanner.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div
+    class="flex flex-col rounded-2xl overflow-hidden relative bg-white shadow-sm"
+  >
+    <div class="min-w-full h-30">
+      <LazyImage
+        :width="1000"
+        :height="1000"
+        alt="will change to api"
+        :url="eventBanner.bannerImg"
+        :placeholder="eventBanner.bannerImgHash"
+        class="object-cover w-full h-full"
+      />
+    </div>
+    <section class="pl-33 flex flex-col pr-6 pt-2 pb-4.5">
+      <h1 class="text-3xl text-blue-10 font-heading mb-1">
+        {{ eventBanner.title }}
+      </h1>
+      <div class="flex flex-wrap mb-2">
+        <base-tag
+          v-for="tag in eventBanner.tags"
+          :key="tag"
+          class="mr-1 mb-1 h-2"
+          >{{ tag }}</base-tag
+        >
+      </div>
+      <section>
+        <div class="flex items-center mb-1">
+          <base-icon
+            width="14px"
+            height="14px"
+            iconColor="#FF855F"
```

Comment:
It seems like we have only a few colors for iconColor. Can we modify base-icon component to accept theme props instead of color to eliminate code redundancy?
![image](https://user-images.githubusercontent.com/12471844/114374064-7173e400-9bad-11eb-9c0d-d9c9e4818eaa.png)

Reply:
NVM, lets focus on more importantn work :)

Reply:
Seems likt @poom-kitti fixed it on dev branch. You can see his chances in `src\commons\UI\user-profile\UserProfile.vue`